drift_dev/lib/src/writer/database_writer.dart

```diff
@@ -117,8 +117,9 @@ class DatabaseWriter {
     // Write List of tables
     final schemaScope = dbScope.leaf();
     schemaScope
-      ..write('@override\nIterable<TableInfo> get allTables => ')
-      ..write('allSchemaEntities.whereType<TableInfo>();\n')
+      ..write(
+          '@override\nIterable<TableInfo<Table, dynamic>> get allTables => ')
+      ..write('allSchemaEntities.whereType<<Table, dynamic>>();\n')
```

Comment:
I think this should be

```suggestion
      ..write('allSchemaEntities.whereType<TableInfo<Table, Object?>>();\n')
```

(using `Object?` over `dynamic` is a preference choice here, either way is fine with me)